Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -28,7 +28,14 @@ static void ResetRumbleLockNeeded();
 static void Reset();
 static void Setup();
 
-static bool s_detected = false;
+enum
```

Comment:
I don’t think I can use a typed enum because I put the status in an int variable (that can contain either one of the two enum values, or a (negative) libusb error code).

Maybe I could replace the enum with a struct wrapping the  status/error code, with `IsDetected`/`IsError`/`GetErrorCode`/`SetDetected`/`SetNotDetected`/`SetErrorCode` methods. What do you think?

Reply:
Oh, feel free to ignore this then.